source/access/mfa_guide.rst

```diff
@@ -208,27 +195,61 @@ on your local machine first.
 
 .. _mfa_client_config:
 
+GUI applications with SSH connection in the background
+------------------------------------------------------
+
+Some applications such as MobaXTerm and FileZilla provide a Graphical User Interface
+(GUI) which makes them very useful when connecting to remote sites. However,
+such an application may not always prompt you to copy/paste the VSC firewall link to
+set up the necessary SSH certificate.
+
+Therefore, one way to connect to VSC is to first connect with an ssh-client 
+on your machine as explained in the previous sections.
+In the case of not having an SSH certificate agent running then as long as you
+keep that connection open you can connect with the other apps as well.
+In case you have already stored the SSH certificate in a running agent
+you can then proceed with connecting to VSC with the application.
+
+.. note::
+
+   Some GUI applications may not always work when connecting to VSC
+   without an agent. For example, NX is one of them.
+   Therefore, it is highly recommended in such cases to use the agent
+   connection method :ref:`ssh agent<mfa_agent>`.
+
 Configuration of ssh-clients and GUI apps
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you have not yet set up your ssh-client or other apps that use ssh to be
 able to use an agent, you might have to make some changes in your connection
-profiles. Different apps will need different changes, but here we shortly show
-what to do for MobaXTerm, PuTTY and NX:
+profiles. Different apps will need different changes. For an explanation
+with screenshots for the various GUI apps please refer to :ref:`mfa quick start`.
+Here below we shortly show what to do for PuTTY, MobaXTerm, NX, and FileZilla:
+
+- PuTTY
```

Comment:
The PuTTY page (https://docs.vscentrum.be/access/text_mode_access_using_putty.html) unfortunately does not talk about Certificates, though it covers agents. 
I guess, at some point in near future, we have to take an initiative to update those pages, and only provide a link to those pages in this MFA document. 
But, this part is OK for now.

Reply:
Oh, I suppose I could also refer to that page. Later we review.

Reply:
I prefer that we do not point to that page for now. It shows SSH keys and people will get confused.